src/main/java/backend/Team.java

```diff
@@ -0,0 +1,318 @@
+package backend;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Team {
+    private String name;
+    private String manager;
+    private int budget;
+    private int pointsAllTime;
+    private int pointsThisSeason;
+    private int winAllTime;
+    private int winThisSeason;
+    private Engine engine;
+    private Aerodynamicist aerodynamicist;
+    private Mechanic mechanic;
+    private Strategist strategist;
+    private List<Driver> drivers;
+
+    /**
+     * Create a new team.
+     * Create a new arraylist for drivers.
```

Comment:
Not necessar to add a comment for that. The reader can see that directly from the code